packages/web/src/utils/commonPasswordCheck.ts

```diff
@@ -23,8 +25,18 @@ export const commonPasswordCheck = async (
   if (result) {
     // @ts-ignore
     const text = (await result?.text()) as string
-    const map = text.split(/\s+/g).map((s) => s.slice(0, s.indexOf(':')))
-    return map.includes(hash.slice(5))
+    const hashArr = text.split(/\s+/g).map((s) => s.slice(0, s.indexOf(':')))
+
+    // If there is no match, return false
+    if (!hashArr.includes(hash.slice(5))) {
```

Comment:
Should we be checking for the full hash instead of just the first 5 characters? Here and on line 35

Reply:
opposite. this is slicing off the first 5 chars and checking the rest. this is just because of how the API returns the data

Reply:
Ah gotcha